docs/src/main/sphinx/admin/properties-sql-environment.md

```diff
@@ -35,6 +35,18 @@ Set the default catalog for [SQL routine](/routines) storage for all clients.
 The connector used in the catalog must support [](sql-routine-management). Any
 usage of a fully qualified name for a routine overrides this default.
 
+The default catalog and schema for SQL routine storage must be configured
+together, and the resulting path must be set as part of the path. For example,
```

Comment:
“the resulting entry”? The path is the combination of schema entries

Reply:
Updated